### DISCUSSION TITLE
Can't add to cart on Bestbuy

### DISCUSSION THREAD

**Discussion by yzd0014** [jef/streetmerchant#2278] [Help]
So I tried a product that is currently in stock on Bestbuy, but the program will only direct me to my chart and it still shows my chart is empty. I wonder if this is an existing problem of the program or there is a parameter to control this. Thank you.

**Comment by Mr-Tech-13** [comment]
I don't think that there is a way to bypass this, or at least from what I can tell

**Reply by yzd0014** [reply]
Do you mean that regardless add to chart is enabled or nor, it doesn't make any difference?

**Reply by Mr-Tech-13** [reply]
I think that BB has some protection against it

**Reply by yzd0014** [reply]
Okay, thanks for letting me know.
